src/test/resources/graphics/scenery/tests/examples/volumes/GridCellsToZero.comp

```diff
@@ -0,0 +1,27 @@
+#version 450
```

Comment:
This file appears twice, one version should be removed.

Reply:
Removed in b93ead334b24bc1b064c5f163eff1b5d771d36f1